contrib/babelfishpg_tsql/src/procedures.c

```diff
@@ -2201,50 +2209,101 @@ gen_sp_droprolemember_subcmds(const char *user, const char *member)
 	return res;
 }
 
-/*
- * Helper function to execute a utility command using
- * ProcessUtility(). Caller should make sure their
- * inputs are sanitized to prevent unexpected behaviour.
- */
-static void
-exec_utility_cmd_helper(char *query_str)
+static void 
+update_bbf_server_options(char *servername, char *querytimeout, bool isInsert)
```

Comment:
we should make the parameter name generic instead of querytimeout since this will be used by other server options later. This method should take 2 arguments option value and option name and based on the option name do the validation and update/insert.